### PR TITLE
Convert Swift client test to Github Actions CI

### PR DESCRIPTION
**Pull request by echeran**

### CODE REVIEW

File: .github/workflows/mynmr_tools_ci.yml [hunk 1]
```diff
@@ -75,3 +75,14 @@ jobs:
...
+  # Swift Client on macOS
+  swift:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Configure XCode version
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: '11.5'
+      - run: make client-swift
```
Comment (sffc) [inline .github/workflows/mynmr_tools_ci.yml:88]:
```suggestion
      - run: make client-swift-test
```

**Review by sffc — CHANGES_REQUESTED** [review]
Thanks!

**Review by sffc — APPROVED** [review]
@echeran Can you resolve the issue with the Google CLA?

**Comment by echeran** [comment]
@sffc My side is now good, it looks like what is left is your unicode.org email address needs to be added to the CLA.

**Pull request merged**